cmd/util/ledger/migrations/utils.go

```diff
@@ -40,6 +35,20 @@ func checkStorageHealth(
 		}
 	}
 
+	return nil
+}
+
+func checkStorageHealth(
+	address common.Address,
+	storage *runtime.Storage,
+	payloads []*ledger.Payload,
+) error {
+
+	err := loadAtreeSlabsInStorge(storage, payloads)
+	if err != nil {
+		return err
+	}
+
 	for _, domain := range allStorageMapDomains {
```

Comment:
Should this be part of `loadAtreeSlabsInStorge`?

Reply:
No because we only need storage maps in Cadence runtime when checking storage map through Cadence.  

Since fixing broken data happens at atree level, we don't need to load storage maps.